CHANGES.next.md

```diff
@@ -575,3 +575,5 @@
   currently limited to the cluster_boot benchmark.
 - Added support for CoreOS Container Linux in GCP, AWS, and Azure Providers.
   Support is currently limited to the cluster_boot benchmark.
+- Fix AliCloud support by using the newer aliyun (instead of aliyuncli) cmd
```

Comment:
Can you move this up to the Breaking Changes section?

Reply:
Done.